foreman.te

```diff
@@ -71,55 +61,82 @@ gen_tunable(websockify_can_connect_all, false)
 
 ## <desc>
 ## <p>
-## Determine whether passenger can connect to Docker via local socket
+## Determine whether Ruby on Rails can connect to Docker via local socket
+## </p>
+## </desc>
+gen_tunable(foreman_rails_can_connect_container_unix, true)
+
+## <desc>
+## <p>
+## Determine whether Ruby on Rails can connect to OpenStack
 ## </p>
 ## </desc>
-gen_tunable(passenger_can_connect_container_unix, true)
+gen_tunable(foreman_rails_can_connect_openstack, true)
 
 ## <desc>
 ## <p>
-## Determine whether passenger can connect to OpenStack
+## Determine whether Ruby on Rails can execute ssh (needed for qemu+ssh libvirt connection).
 ## </p>
 ## </desc>
-gen_tunable(passenger_can_connect_openstack, true)
+gen_tunable(foreman_rails_can_spawn_ssh, true)
 
 ## <desc>
 ## <p>
-## Determine whether passenger can execute ssh (needed for qemu+ssh libvirt connection).
+## Determine whether Ruby on Rails can connect to http_port_t when preloading app.
 ## </p>
 ## </desc>
-gen_tunable(passenger_can_spawn_ssh, true)
+gen_tunable(foreman_rails_can_connect_http, true)
 
 ## <desc>
 ## <p>
-## Determine whether passenger can connect to http_port_t when preloading app.
+## Determine whether Ruby on Rails can connect to puppet_port_t when preloading app.
 ## </p>
 ## </desc>
-gen_tunable(passenger_can_connect_http, true)
+gen_tunable(foreman_rails_can_connect_puppetmaster, true)
 
 ## <desc>
 ## <p>
-## Determine whether passenger can connect to puppet_port_t when preloading app.
+## Determine whether Ruby on Rails can connect to local or remote libvirt.
 ## </p>
 ## </desc>
-gen_tunable(passenger_can_connect_puppetmaster, true)
+gen_tunable(foreman_rails_can_connect_libvirt, true)
 
 ## <desc>
 ## <p>
-## Determine whether passenger can connect to local or remote libvirt.
+## Determine whether Ruby on Rails can connect to remote SMTP services
 ## </p>
 ## </desc>
-gen_tunable(passenger_can_connect_libvirt, true)
+gen_tunable(foreman_rails_can_connect_smtp, true)
 
 ## <desc>
 ## <p>
-## Determine whether passenger can connect to remote SMTP services
+## Deprecated SQLite3 support
 ## </p>
 ## </desc>
-gen_tunable(passenger_can_connect_smtp, true)
+gen_tunable(foreman_rails_can_manage_sqlite, false)
```

Comment:
We've dropped foreman-sqlite3 in 2.1 so you don't need this unless you want to cherry pick this to 2.0.

Reply:
Yeah removing.